Control coverage/Feature/KubernetesService.md

```diff
@@ -3,4 +3,8 @@
\ No newline at end of file
+| <b>ControlId:</b><br>Azure_KubernetesService_Deploy_Enable_Cluster_RBAC<br><b>DisplayName:</b><br>Access policies on Notification Hub must not have Manage access permissions.<br><b>Description: </b><br> Access policies on Notification Hub must not have Manage access permissions. | <b> ARM API to list Container Services at subscription level: </b> <br> /subscriptions/{subscriptionId}/providers/Microsoft.ContainerService/managedClusters?api-version=2020-09-01 <br><b>Properties:</b><br> properties.enableRBAC | <b>Passed: </b><br> RBAC is enabled for AKS.<br><b>Failed: </b><br>RBAC is disabled for AKS. |
+|<b>ControlId:</b><br>Azure_KubernetesService_Audit_Enable_Monitoring<br><b>DisplayName:</b><br>Monitoring must be enabled for Azure Kubernetes Service.<br><b>Description: </b><br>Monitoring must be enabled for Azure Kubernetes Service. | <b> ARM API to list Container Services at subscription level: </b> <br> /subscriptions/{subscriptionId}/providers/Microsoft.ContainerService/managedClusters?api-version=2020-09-01<br><b>Properties:</b><br> properties.addonProfiles.omsagent.enabled  | <b>Passed: </b><br> Kubernetes resource have LA configured and monitoring is enabled. <br><b>Failed: </b><br>Kubernetes resource do not have LA configured and monitoring  is disabled.|
+|<b>ControlId:</b><br>Azure_KubernetesService_DP_Disable_HTTP_Application_Routing	<br><b>DisplayName:</b><br>HTTP application routing should be disabled in Kubernetes Service.<br><b>Description: </b><br>HTTP application routing should be disabled in Kubernetes Service. | <b> ARM API to list Container Services at subscription level: </b> <br> /subscriptions/{subscriptionId}/providers/Microsoft.ContainerService/managedClusters?api-version=2020-09-01<br><b>Properties:</b><br> properties.addonProfiles.httpApplicationRouting.enabled| <b>Passed: </b><br> HTTP app routing is disabled for Kubernetes resource. <br><b>Failed: </b><br>HTTP app routing is enabled for Kubernetes resource.|
+|<b>ControlId:</b><br>Azure_KubernetesService_NetSec_Dont_Open_Management_Ports<br><b>DisplayName:</b><br>Do not leave management ports open on Kubernetes nodes.<br><b>Description: </b><br>Do not leave management ports open on Kubernetes nodes unless required. | <b> ARM API to list Container Services at subscription level: </b> <br> /subscriptions/{subscriptionId}/providers/Microsoft.ContainerService/managedClusters?api-version=2020-09-01<br> | <b>Passed: </b><br> NSG is configured and no inbound port is open or if restricted ports not found.<br><b>Failed: </b><br>NSG is not configured or if restricted ports found.<br><b>NotApplicable</b>Control is not applicable for ERvNET connected AKS VM/VMSS:|
```

Comment:
No properties are mentioned for this control, Also there are typos in Control speclet column.